src/SprykerEco/Client/Computop/ComputopClientInterface.php

```diff
@@ -52,6 +52,18 @@ public function saveSofortInitResponse(QuoteTransfer $quoteTransfer);
      */
     public function saveIdealInitResponse(QuoteTransfer $quoteTransfer);
 
+    /**
+     * Specification:
+     * - Saves PayU CEE Single response to DB
```

Comment:
Saves PayU CEE Single response to the database.

Reply:
Done